tests/album-create.test.js

```diff
@@ -0,0 +1,35 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const db = require('../src/db');
+const app = require('../src/app');
+
+describe('create Album', () => {
```

Comment:
You have an eslint error here. This is happening because VSCode doesnt know that you project uses jest. This can be fixed by adding a rule to your `.eslintrc.js`, see that file for the rule to add. If the error still stays in the test file after adding this you may have to restart your IDE for the change to be registered and the error to disappear